tamr_unify_client/models/project/estimated_pair_counts.py

```diff
@@ -0,0 +1,42 @@
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class EstimatedPairCounts(BaseResource):
+    """Estimated Pair Counts info for Mastering Project"""
+
+    @classmethod
+    def from_json(cls, client, resource_json, api_path=None) -> "EstimatedPairCounts":
+        return super().from_data(client, resource_json, api_path)
+
+    @property
+    def is_up_to_date(self) -> bool:
+        """Whether an estimate pairs job has been run since the last edit to the binning model.
+
+        :type: bool
+        """
+        return self._data.get("isUpToDate")
+
+    @property
+    def total_estimate(self) -> dict:
+        """The total number of estimated candidate pairs and generated pairs for the model across all clauses.
+
+        :type: dict
```

Comment:
Please specify key/value types for `dict`. E.g. if you are expecting `str` keys and `int` values: `:type: dict[str, int]`.

See [this example in our codebase](https://github.com/Datatamer/unify-client-python/blob/master/tamr_unify_client/models/dataset/resource.py#L39) and [how it renders nicely in our docs](https://tamr-unify-python-client.readthedocs.io/en/latest/developer-interface.html#tamr_unify_client.models.dataset.resource.Dataset.tags).